layabase/_database_sqlalchemy.py

```diff
@@ -411,6 +415,27 @@ def _get_required_query_fields(cls) -> List[str]:
             if marshmallow_field.metadata.get("required_on_query", False)
         ]
 
+    @classmethod
+    def _check_choices_query_fields(cls, row):
+        choice_options, name = cls._get_choices_query_fields()
```

Comment:
doing this operation every time seems time consuming, can you do it at max once per request? Bonus point to do it only once per python process lifetime ?